charts/neon-proxy/values.yaml

```diff
@@ -119,6 +119,8 @@ settings:
   redisUserId: "neon"
   # settings.awsRegion -- (string) Aws region to retrieve credentials
   awsRegion: ""
+  # settings.proxyProtocolV2 -- (string) Whether to enable or disable the parsing of PROXY protocol V2. "rejected", "supported", or "required" are accepted.
+  proxyProtocolV2: ""
```

Comment:
The flag is unconditionally set in the yaml template, so I think we can leave an empty string here, right?

Reply:
the empty string is the conditional (yay go templating)

Reply:
TIL

```
{{with pipeline}} T1 {{end}}
	If the value of the pipeline is empty, no output is generated;
	otherwise, dot is set to the value of the pipeline and T1 is
	executed.
```

Reply:
yeah I need to consult the docs everytime. helm is great